public/js/event_admin_list_script.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+$(".entity-delete").click(function () {
+    return confirm('Вы уверены, что хотите удалить эту запись?');
+});
+
+$(".event-cancel").click(function () {
+    return confirm('Вы уверены, что хотите изменить статус этого события на "отменено"? Событие перестанет отображаться в списке для пользователей, но не будет удалено из базы и гугл-календаря.');
+});
+
+$(".event-approve").click(function () {
+    return confirm('Вы уверены, что хотите опубликовать это событие на сайте и в гугл-календаре?');
+});
+
+$(".event-restore").click(function () {
```

Comment:
@daria-sieroshtan 
You can leave it as is for simplicity, but generally it's a bad practice to put in *general* modules stuff which only makes sense for *specific* pages